bin/runner.js

```diff
@@ -108,6 +60,7 @@ function generate(model, allTestsGenerated) {
     "tests"
   );
   var testsDocPath = path.join(model.output, "Doc");
+  rimraf.sync(testsDocPath);
```

Comment:
Probably worth doing some checks that the path are what we expect before running this

Reply:
what do you want to check?